platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -236,14 +248,15 @@ could publish a form for users to submit feature requests; and conduct periodic
 roadmap meetings to share upcoming features and gather feedback.
 
 A platform team doesn't necessarily run compute, network, storage or other
-services. Rather they own the interfaces \[2\] (GUI, CLI, API) and user
-experiences with those services. For example, a Web portal page might describe
-and even offer a button to provision an identity for an app; while the
-implementation of that capability might be via a cloud-hosted identity service.
-A platform team owns the web page and user-facing API, but not necessarily the
-implementation.
-
-Earlier it was stated that a platform should be treated as a product.
+services, and in fact an internal platform should be as thin as viable as
+described above. A platform team instead is most responsible for the
+_interfaces_ \[2\] (GUI, CLI, API) and user experiences with those services. For
+example, a Web portal page might describe and even offer a button to provision
+an identity for an app; while the implementation of that capability might be via
+a cloud-hosted identity service.  An internal platform team may manage the web
+page and user-facing API, but not necessarily the implementation.
```

Comment:
```suggestion
services, and in fact an internal platform should maintain a minimal footprint as
described above. A platform team instead is most responsible for the
_interfaces_ \[2\] (GUI, CLI, API) and user experiences with those services. For
example, a Web portal page might describe and even offer a button to provision
an identity for an app; while the implementation of that capability might be via
a cloud-hosted identity service.  An internal platform team may manage the web
page and user-facing API, but not necessarily the implementation. Only when a 
required capability is not available in the market should a platform team create 
and manage its complexity in house.
```

Reply:
Took a shot at updating in latest commit, PTAL!